src/main/java/io/aiven/kafka/auth/json/UserPassword.java

```diff
@@ -0,0 +1,26 @@
+package io.aiven.kafka.auth.json;
```

Comment:
Please add the header (with the current year).

Reply:
will fix it